addons/nightvision/CfgWeapons.hpp

```diff
@@ -5,6 +5,14 @@ class CfgWeapons {
         modelOptics = "";
         GVAR(border) = QPATHTOF(data\nvg_mask_binos_4096.paa);
         GVAR(bluRadius) = 0.15;
+        // It's should prevent from main classes incorrectly work (Set Color Correction Params) / Original ACE3's Green (JDT & AleM)
```

Comment:
`It's should prevent from main classes incorrectly work` -> `This prevents the main classes from working incorrectly`
But might aswell leave that comment away.  This is setting the NVG Color. No need for a comment besides maybe `//Green color`

Reply:
Well, i try'd to be more as informative as possible. Ofc, it would be changed.